src/main/resources/db/migration/v0.38.0.3__gaf-load.sql

```diff
@@ -0,0 +1,3 @@
+update bulkurlload set bulkloadurl = 'https://download.alliancegenome.org/2.3.0/GAF/HUMAN/GAF_HUMAN_1.gaf'
```

Comment:
Need to bump the version of this file

Reply:
This needs to be turned into a FMS load which will always pull the latest version of the file.

Reply:
Do we know from where this file got downloaded? Only the human file is referenced from the FMS in the YAML file:
https://github.com/alliance-genome/agr_ferret/blob/master/src/datasets/GAF.yaml